.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,15 @@
+# Ref: https://help.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser
+blank_issues_enabled: true  # default
+contact_links:
+- name: '💬 IRC: #pypa @ Freenode'
+  url: https://webchat.freenode.net/#pypa
+  about: Chat with devs.
+- name: 🤷💻🤦 Discourse
+  url: https://discuss.python.org/c/packaging
+  about: Please ask typical Q&A here.
+- name: 🔐 Security Policy
+  url: https://pypi.org/security/
+  about: Please learn how to report security vulnerabilities here.
+- name: 📝 PyPA Code of Conduct
```

Comment:
Instead of putting this here, I would rather add our Code of Conduct per the [adding a custom Code of Conduct instructions here](https://help.github.com/en/github/building-a-strong-community/adding-a-code-of-conduct-to-your-project#adding-a-code-of-conduct-manually). GitHub will use it in various calls-to-action and notifications for new or returning contributors which will be more widely seen than this link. This activity could be done in a separate PR.

Reply:
I think it's better to do this not instead of but additionally to putting a link. Also, GitHub is not good at recognizing custom templates and external references. Plus there was a discussion (probably on discuss?) about using https://github.com/pypa/.github for this because it's supposed to be the same across repos.

Reply:
OK, that's fair.